views/components/Sidebar/NavItem.jsx

```diff
@@ -5,10 +5,16 @@ import SidebarLink from './SidebarLink'
 const NavItem = ({ to, Icon, label }) => {
   return (
     <ListItem component={SidebarLink} to={to}>
-      <ListItemIcon sx={{ minWidth: 40, alignSelf: 'center' }}>
-        <Icon />
+      <ListItemIcon
+        sx={{ minWidth: 20, width: 20, height: 20, alignSelf: 'center' }}
+      >
+        <Icon sx={{ height: 20, width: 20 }} />
```

Comment:
```suggestion
        sx={{ minWidth: 20, width: 20, height: 20, alignSelf: 'center' }}
      >
        <Icon sx={{ height: 20, width: 20 }} />
```
Do we need to repeat the width in both places..? do we need height and width in the ListItemIcon..?

Reply:
yeah we need it in this case so that the icon and the wrapper around it are the same size. Helps us control the spacing between this element and the text.

Reply:
Okay, you recommended using padding, is this a case where we can use that..?

Reply:
No we don’t want to use padding in this case because we want these exact sized icons. Padding is useful for spacing inside elements, margin for outside elements.